Jenkinsfile

```diff
@@ -20,7 +19,9 @@ pipeline {
         booleanParam(name: 'MOCK_MVC_TESTS', defaultValue: true, description: 'Run Mock MVC tests')
         booleanParam(name: 'INTEGRATION_TESTS', defaultValue: true, description: 'Run Integration tests')
         booleanParam(name: 'DEGRADED_TESTS', defaultValue: true, description: 'Run degraded mode tests')
-        booleanParam(name: 'PUSH_TO_BUILD_GE', defaultValue: false, description: 'Publish to build artifactory')
```

Comment:
There are 3 separate types of "upload artifacts" in UAA and ACS:

- Build.GE artifactory upload
- Maven Central upload
- Bintray upload

It's better to have the variable explicitly mention "Build GE" rather than a generic "upload".

Reply:
Also make sure that this variable name is consistent with other projects.

Reply:
Sure, can revert. I didn't really know what BUILD_GE meant